src/clj/athens/self_hosted/event_log.clj

```diff
@@ -6,12 +6,28 @@
     [clojure.edn :as edn]
     [clojure.string :as str]
     [clojure.tools.logging :as log]
-    [fluree.db.api :as fdb]))
+    [fluree.db.api :as fdb])
+  (:import
+    (java.util
+      UUID)))
 
 
 (def ledger "events/log")
 
 
+(defn create-fluree-comp
```

Comment:
Can you move this function to `athens.self-hosted.components.fluree` instead, and also use it there inside the `defrecord` `start` method?